packages/rum-core/src/domain/pageActivityObservable.ts

```diff
@@ -10,32 +10,20 @@ export interface PageActivityEvent {
   isBusy: boolean
 }
 
-export type CompletionCallbackParameters = { hadActivity: true; endClocks: ClocksState } | { hadActivity: false }
+export type PageActivityCompletionEvent = { hadActivity: true; endClocks: ClocksState } | { hadActivity: false }
 
-export function waitIdlePageActivity(
+export function createIdlePageActivityObservable(
```

Comment:
should not the file be `idlePageActivityObservable.ts` then?

Reply:
true